packages/vuetify/src/labs/VStepper/VStepper.tsx

```diff
@@ -0,0 +1,185 @@
+    default: 'title',
+  },
+  itemValue: {
+    type: String,
+    default: 'value',
+  },
+  mobile: Boolean,
+  nonLinear: Boolean,
+  flat: Boolean,
+  showActions: Boolean,
+
+  ...makeGroupProps({
+    mandatory: 'force' as const,
+    selectedClass: 'v-stepper-item--selected',
+  }),
+  ...omit(makeVSheetProps(), ['color']),
+  ...makeVStepperActionsProps(),
+}, 'VStepper')
+
+export const VStepper = genericComponent<VStepperSlots>()({
+  name: 'VStepper',
+
+  props: makeVStepperProps(),
+
+  emits: {
+    'update:modelValue': (v: unknown) => true,
+  },
+
+  setup (props, { slots }) {
+    const { next, prev } = useGroup(props, VStepperSymbol)
+
+    const items = computed(() => props.items.map((item, index) => {
+      const title = getPropertyFromItem(item, props.itemTitle, item)
+      const value = getPropertyFromItem(item, props.itemValue, index + 1)
+
+      return {
+        title,
+        value,
+        raw: item,
+      }
+    }))
+
+    const slotProps = computed(() => ({ next, prev }))
+
+    provideDefaults({
+      VStepperItem: {
+        editable: computed(() => props.editable),
+        prevText: computed(() => props.prevText),
+        nextText: computed(() => props.nextText),
+      },
+    })
+
+    useRender(() => {
+      const [sheetProps] = VSheet.filterProps(props)
+      const [stepperActionProps] = VStepperActions.filterProps(props)
+
+      const hasHeader = !!(slots.header || props.items.length)
+      const hasWindow = props.items.length > 0
+      const hasActions = props.showActions && !!(hasWindow || slots.actions)
+
+      return (
+        <VSheet
+          { ...sheetProps }
+          class={[
+            'v-stepper',
+            {
+              'v-stepper--alt-labels': props.altLabels,
+              'v-stepper--flat': props.flat,
+              'v-stepper--non-linear': props.nonLinear,
+              'v-stepper--mobile': props.mobile,
+            },
+            props.class,
+          ]}
+          style={ props.style }
+        >
+          { hasHeader && (
+            <VStepperHeader key="stepper-header">
+              { items.value.map((item, index) => (
+                <>
+                  { !!index && (<VDivider />) }
+
+                  <VStepperItem
+                    { ...item }
+                    v-slots={{
+                      default: slots.item,
+                      icon: slots.icon,
+                      title: slots.title,
+                      subtitle: slots.subtitle,
+                    }}
+                  />
+                </>
+              ))}
+
+              { slots.header?.() }
+            </VStepperHeader>
+          )}
+
+          { hasWindow && (
+            <VStepperWindow key="stepper-window">
+              { items.value.map((item, index) => {
+                return (
+                  <VStepperWindowItem
+                    value={ item.value }
+                    v-slots={{ default: slots[`window-item.${item.value}`] }}
```

Comment:
`window-item.x` is very wordy. I think it would be better to switch the name with the header slot. So that content is `item.x` and header is `header-item.x` or something similar. My assumption is that it will be a lot more common to have content slot than header slot.